src/NzbDrone.Core/Indexers/Definitions/Redacted.cs

```diff
@@ -42,13 +71,290 @@ protected override IndexerCapabilities SetCapabilities()
+                        PublishDate = DateTimeOffset.FromUnixTimeSeconds(result.GroupTime).UtcDateTime,
+                        Freeleech = result.IsFreeLeech || result.IsPersonalFreeLeech,
+                        Files = result.FileCount,
+                        Grabs = result.Snatches,
+                    };
+
+                    var category = result.Category;
+                    if (category == null || category.Contains("Select Category"))
+                    {
+                        release.Categories = _categories.MapTrackerCatToNewznab("1");
+                    }
+                    else
+                    {
+                        release.Categories = _categories.MapTrackerCatDescToNewznab(category);
+                    }
+
+                    torrentInfos.Add(release);
+                }
+            }
+
+            // order by date
+            return
+                torrentInfos
+                    .OrderByDescending(o => o.PublishDate)
+                    .ToArray();
+        }
+
+        private string GetDownloadUrl(int torrentId, bool canUseToken)
+        {
+            // AuthKey is required but not checked, just pass in a dummy variable
+            // to avoid having to track authkey, which is randomly cycled
+            var url = new HttpUri(_settings.BaseUrl)
+                .CombinePath("/torrents.php")
+                .AddQueryParam("action", "download")
+                .AddQueryParam("id", torrentId)
+                .AddQueryParam("authkey", "prowlarr")
+                .AddQueryParam("torrent_pass", _settings.Passkey)
+                .AddQueryParam("usetoken", (_settings.UseFreeleechToken && canUseToken) ? 1 : 0);
+
+            return url.FullUri;
+        }
+
+        private string GetInfoUrl(string groupId, int torrentId)
+        {
+            var url = new HttpUri(_settings.BaseUrl)
+                .CombinePath("/torrents.php")
+                .AddQueryParam("id", groupId)
+                .AddQueryParam("torrentid", torrentId);
+
+            return url.FullUri;
+        }
+    }
+
+    public class RedactedSettingsValidator : AbstractValidator<RedactedSettings>
+    {
+        public RedactedSettingsValidator()
+        {
+            RuleFor(c => c.Apikey).NotEmpty();
+        }
+    }
+
+    public class RedactedSettings : IIndexerSettings
+    {
+        private static readonly RedactedSettingsValidator Validator = new RedactedSettingsValidator();
+
+        public RedactedSettings()
+        {
+            Apikey = "";
+            Passkey = "";
+            UseFreeleechToken = false;
+        }
+
+        [FieldDefinition(1, Label = "Base Url", Type = FieldType.Select, SelectOptionsProviderAction = "getUrls", HelpText = "Select which baseurl Prowlarr will use for requests to the site")]
+        public string BaseUrl { get; set; }
+
+        [FieldDefinition(2, Label = "API Key", HelpText = "Redacted API Key")]
```

Comment:
standardize to follow pattern

E.g. for BHD `Label = "API Key", HelpText = "API Key from the Site (Found in My Security => API Key)", Privacy = PrivacyLevel.ApiKey)]`

Reply:
Good catch, done.